### PR TITLE
Hue correction

### PR DESCRIPTION
**Pull request by andreasbrett**
Implementation of https://github.com/Koenkk/zigbee2mqtt/issues/4026

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -2130,7 +2139,7 @@ const converters = {
             if (key === 'state') {
                 await entity.read('genOnOff', ['onOff']);
             } else if (key === 'color') {
-                await entity.read('lightingColorCtrl', ['currentHue', 'currentSaturation']);
+                await entity.read('lightingColorCtrl', ['currenthHue', 'currentSaturation']);
```
Comment (sjorge) [inline converters/toZigbee.js:2142]: Are you sure this change is correct?

Comment (andreasbrett) [reply converters/toZigbee.js:2142]: yes, it had been merged some hours ago already. https://github.com/Koenkk/zigbee-herdsman-converters/pull/1450

Comment (andreasbrett) [reply converters/toZigbee.js:2142]: Sorry, reintroduced that although I PRed this some hours ago... https://github.com/Koenkk/zigbee-herdsman-converters/pull/1450

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**